packages/stream-model-instance-handler/src/model-instance-document-handler.ts

```diff
@@ -218,7 +217,8 @@ export class ModelInstanceDocumentHandler implements StreamHandler<ModelInstance
     ceramic: StreamReader,
     model: Model,
     content: any,
-    genesis: boolean
+    genesis: boolean,
+    payload?: any
```

Comment:
Could you type the `payload` here please? I think it should be an array of `Operation` from https://github.com/Starcounter-Jack/JSON-Patch/blob/master/src/core.ts#L13

Reply:
jsonpatch doesnt export the Operation type so i created an equivalent one to be able to type the payload argument.